#:===  ur-robot params :===
# command_ur10_topic: "/ur_driver/URScript"
read_wrench_topic: "/optoforce_node/wrench_HEXEA152" # /wrench
# read_toolVel_topic: "/tool_velocity"
# read_jointState_topic: "/joint_states"
base_frame: "base"
tool_frame: "tool0_controller" # "optoforce_frame" #"tool0_controller"
use_sim_time: false
host: "localhost"
reverse_port: 50001
